eks-anywhere-common/Testers/Kong/test-job-configmap.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-configmap
+  namespace: kong
+data:
+  test.sh: |-
+    #!/bin/bash
+    kubectl apply -f https://bit.ly/echo-service
+    echo "
+    apiVersion: networking.k8s.io/v1
+    kind: Ingress
+    metadata:
+      name: demo
+    spec:
+      ingressClassName: kong
+      rules:
+      - http:
+          paths:
+          - path: /foo
+            pathType: ImplementationSpecific
+            backend:
+              service:
```

Comment:
I see you are creating an Ingress and trying to curl to an ep which points to `echo` service but unclear on how this is related to functional test to confirmation of Kong gateway is working fine?